### DISCUSSION TITLE
add move/delete to right click popup

### DISCUSSION THREAD

**Discussion by pcause** [LostPaul/obsidian-folder-notes#235] [Ideas]
when i right click a note name in the folder note I get a popup that has several useful function.  It would be helpful to add move and delete to the capabilities.  This would make reorganizing information much quicker/easier.

**Comment by LostPaul** [comment]
> when i right click a note name in the folder note I get a popup that has several useful function. It would be helpful to add move and delete to the capabilities. This would make reorganizing information much quicker/easier.

Delete the folder note file or what? For moving I'll be adding a command to unlink a folder note from a folder soon but until then you can use the exclude folders feature or make other turn other existing notes into folder notes with "Turn into folder note".

**Comment by pcause** [comment]
delete the file that is in the folder note that I clicked.  So assume a folder note looks like:

a
b
c

I right click b.  I get a popup menu.  It had delete as an entry.  I select delete and the file b (b.md) is deleted from the vault/filesystem.

**Comment by LostPaul** [comment]
I still don't understand what you exactly want? What is the current delete option in the popup missing and what should the "move" do? Only move the folder note file alone or also move the folder along with it?
